common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -294,12 +294,20 @@ status_function() {
     echo "$0: status"
 
     pgrep -l beerocks
-    pgrep -l ieee1905_transport
```

Comment:
please provide a description about this change in the commit message

Reply:
This is actually removed by mistake. Good cache!.
retune the line.